aip/0009.md

```diff
@@ -0,0 +1,99 @@
+---
+aip:
+  id: 9
+  state: draft
+  created: 2019-08-01
+  modified: 2019-08-04
+permalink: /9
+redirect_from:
+  - /09
+  - /009
+  - /0009
+---
+
+# AIP Glossary
+
+In the name of brevity, this AIP defines some common terminology here rather
+than in each AIP individually.
+
+## Guidance
+
+The following terminology **should** be used consistently throughout AIPs.
+
+API
+: Application Programming Interface. This can be a local interface (such as a
+  client library) or a Network API (defined below).
+
+Network API
+: An API that operate across a network of computers. Network APIs communicate
+  using network protocols including HTTP, and are frequently produced by
+  organizations separate from those that consume them.
+
+Google API
+: A Network API exposed by a Google service. Most of these are hosted on the
+  googleapis.com domain. It does not include other types of APIs, such as client
+  libraries and SDKs.
+
+API Definition
+: The defintion of an API, usually defined in a Protocol Buffer service. It is
+  typically mapped to an interface in most programming languages. An API
+  Definition can be implemented by any number of API Services.
+
+API Version
+: The version of an API or a group of APIs if they are defined together. An API
+  Version is often represented by a string, such as "v1", and presents in API
+  requests and Protocol Buffers package names.
```

Comment:
We'll need to work out whether the stability should be part of the version string when we talk about it.

Reply:
OK - leaving this as is for now.